src/nginxconfig/templates/domain_sections/https.vue

```diff
@@ -285,6 +334,20 @@ THE SOFTWARE.
                 },
                 deep: true,
             },
+            // Only allow port reuse when HTTP/3 is enabled first
+            '$props.data.http3': {
+                handler(data) {
+                    // This might cause recursion, but seems not to
+                    if (data.computed) {
+                        this.$props.data.portReuse.enabled = true;
+                        this.$props.data.portReuse.computed = this.$props.data.portReuse.value;
+                    } else {
+                        this.$props.data.portReuse.enabled = false;
+                        this.$props.data.portReuse.computed = false;
+                    }
+                },
+                deep: true,
+            },
```

Comment:
Why is this the case? reuseport is completely separate of http2/3?

Reply:
Nvm, ignore me on this, trying to do this is probably going to result in a more complex UI that'll confuse folks.

Reply:
I thought they did but apparently not cant find any docs denying or approving it, so /shrug